backend/src/script.ts

```diff
@@ -32,4 +34,37 @@ const formatReview = (data: any): Review => ({
   photos: data.photos,
 });
 
+const formatLandlord = ({
+  id,
+  name,
+  contact,
+  avgRating,
+  photos,
+  reviews,
+  properties,
+}: // eslint-disable-next-line @typescript-eslint/no-explicit-any
+any): LandlordWithId => ({
+  id: id.toString(),
+  name,
+  contact,
+  avgRating: avgRating.length > 0 ? Number.parseFloat(avgRating) : 0,
+  photos: photos.length > 0 ? photos : [],
+  reviews: reviews.length > 0 ? reviews : [],
+  properties: properties.length > 0 ? properties : [],
+});
+
+const makeLandlord = async (landlordWithId: LandlordWithId) => {
+  try {
+    const { id, ...rest } = landlordWithId;
+    const doc = landlordCollection.doc(id);
+    const landlord = rest as Landlord;
+    doc.set({ ...landlord });
+  } catch (err) {
+    // eslint-disable-next-line no-console
```

Comment:
The eslint config for `/backend` has `no-console` turned off so there is no reason to disable it anywhere.